.vib/mongodb/goss/goss.yaml

```diff
@@ -1,33 +1,36 @@
-process:
-  mongod:
-    running: true
+file:
+  /bitnami/mongodb:
```

Comment:
I think we should add `persistence.mountPath` to `runtime_params` and the vars file and parametrize this.

Reply:
Done, thanks.